roles/bpftrace/README.md

```diff
@@ -38,8 +38,8 @@ Make bpftrace metrics available to PCP analysis tools, and allow the local *graf
     - role: performancecopilot.metrics.bpftrace
       vars:
         bpftrace_users:
-        - { user: 'grafana' }
-        - { user: 'metrics', sasluser: 'metrics', saslpassword: 'p4ssw0rd' }
+          - {user: 'grafana'}
+          - {user: 'metrics', sasluser: 'metrics', saslpassword: 'p4ssw0rd'}
```

Comment:
Out of curiosity, in which case are we supposed to keep single quotes vs. eliminate them?

Reply:
https://github.com/redhat-cop/automation-good-practices/tree/main/coding_style#yaml-and-jinja2-syntax
```
Do not use quotes unless you have to, especially for short module-keyword-like strings like present, absent, etc. But do use quotes for user-side strings such as descriptions, names, and messages.
```
In this case, `{user: 'grafana'}` - I would have to refer to the YAML standard to see if `{user: 'grafana'}` is ok, since it is a JSON construct embedded within a YAML document.

Reply:
I see. Thanks!